srcpkgs/nvme-cli/template

```diff
@@ -18,9 +18,15 @@ checksum=d7c3d56f36f81c0933f7b7c3f90c4196be0fec758affc8062c525b6bc5ea65f5
 
 CFLAGS="-Wno-error -I."
 
-# these files are touch'd by install, remove so they don't get registered
-# by the package (which would subsequently cause trouble during removal)
 post_install() {
+	# these files are touch'd by install, remove so they aren't registered
+	# by the package (which would cause trouble during removal)
 	rm -f "${PKGDESTDIR}/etc/nvme/hostnqn"
 	rm -f "${PKGDESTDIR}/etc/nvme/hostid"
+	# no need for systemd files
+	systemd="${PKGDESTDIR}/usr/lib/systemd"
+	rm -f "${systemd}/system/nvmf-connect.target"
+	rm -f "${systemd}/system/nvmf-connect@.service"
+	rm -f "${systemd}/system/nvmefc-boot-connections.service"
+	rmdir "${systemd}/system" "${systemd}"
```

Comment:
- the quoting is unnecessary
- PKGDESTDIR == DESTDIR in the main pkg

Reply:
also, just do `rm -rf /foo`

Reply:
While not mandatory this time, it's good habit to quote variables.

Reply:
@Chocimier I believe that quotes are not needed if you use the ${var} semantics

Reply:
^ thats not right.

Reply:
I used PKGDESTDIR to agree with existing post_install commands and removed each file because I figured explicit was better than implicit. I've now changed PKGESTDIR to DESTDIR everywhere and just remove /usr/lib/systemd in one recursive shot. I've kept the quotes in case DESTDIR magically grows a space in the future.